snapcraft/file_utils.py

```diff
@@ -150,3 +150,18 @@ def _search_and_replace_contents(file_path, search_pattern, replacement):
             f.seek(0)
             f.truncate()
             f.write(replaced)
+
+
+def remove_files(directory, remove_pattern_func=lambda file_name: True):
+    """Remove files from directory if remove_pattern_func returns True."""
+    for root, _, files in os.walk(directory):
+        for file_name in files:
+            if remove_pattern_func(file_name):
```

Comment:
Do you think there's a benefit to passing the entire path to the `remove_pattern_func()`?